components/pagination/examples/00-default.js

```diff
@@ -46,13 +46,11 @@ function Example({ brand }) {
 				back={{
 					visible: true,
 					text: 'Go back',
-					assistiveText: page => `Go to previous page which is ${page}`,
```

Comment:
Can we not remove the examples? Rather let's make them better or people don't know about the API?!

Reply:
In the intopia review notes, they said that the text used for the aria-label in the back/next buttons should be matching with the actual button text. So I changed the code to reflect this and therefore the assistiveText option isnt being used anymore.

Reply:
oh... I see.. we removed it completely? ok cool ... I guess